tables/rx_potency/chen21e-pot.csv

```diff
@@ -0,0 +1,118 @@
+ref_name,rx_name,iso_name,section,assay_name,potency_type,potency,cumulative_count,potency_upper_limit,potency_lower_limit,potency_unit,date_added
+Chen21e,COV2-2130,D614G,Figure 1c,Virus isolate,IC50,26,1,10000,1,ng/ml,1/8/2022
+Chen21e,COV2-2130,N501Y/D614G,Figure 1c,Virus isolate,IC50,33,1,10000,1,ng/ml,1/8/2022
+Chen21e,COV2-2130,B.1.1.7,Figure 1c,Virus isolate,IC50,10,1,10000,1,ng/ml,1/8/2022
```

Comment:
I am unable to comment in `isolate_mutations.csv` before the format issue is fixed. However, we don't use `B.1.1.7` as the isolate name. For this case, let's prepend the RefID to the isolate name:

- `Chen21e-B.1.1.7`
- `Chen21e-B.1.351`
- `Chen21e-B.1.1.28`
- `Chen21e-B.1.429`
- `Chen21e-B.1.617.1`
- `Chen21e-B.1.526/S477N`
- `Chen21e-B.1.526/E484K`